README.md

```diff
@@ -19,3 +15,70 @@ To run this application, you need to first clone or download the ‘Create a Toa
+
+<hr style="height:0.3px;border:none;color:lightgrey;background-color:lightgrey;" />
+
+<p align="center">
+<a href="mailto:sales@syncfusion.com?Subject=Syncfusion Blazor toast - GitHub" target="_top">sales@syncfusion.com</a> | <a href="https://www.syncfusion.com?utm_source=github&utm_medium=listing&utm_campaign=blazor-toast-github-samples">www.syncfusion.com</a>| Toll Free: 1-888-9 DOTNET <br>
```

Comment:
Change this as **Syncfusion Blazor Toast - GitHub**
![image](https://user-images.githubusercontent.com/61148953/179900274-4f9336c5-96c6-40a4-8ad1-1b003f28df2d.png)

Reply:
changed